isotovideo

```diff
@@ -298,7 +293,7 @@ $command_handler = OpenQA::Isotovideo::CommandHandler->new(
     cmd_srv_fd => $cmd_srv_fd,
     backend_fd => $backend_process->channel_in,
 );
-$command_handler->on(tests_done => sub {
+$command_handler->on(tests_done => sub (@) {
```

Comment:
```suggestion
$command_handler->on(tests_done => sub ($e, @args) {
```

c.f. https://metacpan.org/pod/Mojo::EventEmitter#on

Reply:
I would name it as this is the officially documented signature. The additional information doesn't hurt. There are no "unused variable" warnings in Perl. In other languages where you would get a warning it is even practice to use things like [Q_UNUSED](https://doc.qt.io/qt-5/qtglobal.html#Q_UNUSED).